spec/lib/sentry/processor/filter_request_body_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Sentry::Processor::FilterRequestBody do
+  let(:client) { double('client') }
+  let(:processor) { Sentry::Processor::FilterRequestBody.new(client) }
+  let(:result) { processor.process(data) }
+
+  let(:data) do
+    {
+      'tags' => { 'controller_name' => 'ppiu', 'sign_in_method' => { 'service_name' => 'idme', 'acct_type' => nil } },
+      'request' =>
+       {
+         'data' =>
+          "{\n  \"account_type\": \"Checking\"}"
+       }
+    }
+  end
+
+  def self.expect_filter(filtered)
+    it "should#{filtered ? '' : "n't"} filter the request body" do
+      expect(result['request']['data']).public_send(
+        filtered ? 'to' : 'to_not',
+        eq(Sentry::Processor::PIISanitizer::FILTER_MASK)
+      )
+    end
+  end
+
+  context 'with data from ppiu' do
+    expect_filter(true)
+  end
+
+  context 'with data from another controller' do
+    before do
```

Comment:
There's only one example here -- let's just set the data

Reply:
i don't think it's a bad idea to have a context with only one example, rspec itself has many specs like that https://github.com/rspec/rspec-core/blob/master/spec/rspec/core/formatters_spec.rb#L78

Reply:
These two specs have done line that requires a fair bit of parsing to understand what's going on. I'd like to see specs that are clear. I can live with the context. I don't like that the whole spec relies on mentally untangling a lot of other code when the actual spec only is a few lines long.